airbyte/logs.py

```
@@ -15,14 +15,16 @@
 import os
 import platform
 import tempfile
+from time import strptime
```

Ruff (0.8.2)

`time.strptime` imported but unused

Remove unused import: `time.strptime`

(F401)